<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartExportDialog</class>
 <widget class="QDialog" name="ChartExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chart Export Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,1">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Image size</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="imageWidth">
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="imageWidthUnit">
          <item>
           <property name="text">
            <string>cm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>inches</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="imageHeight">
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="imageHeightUnit">
          <item>
           <property name="text">
            <string>cm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>inches</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Resolution:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QComboBox" name="imageResolution">
          <item>
           <property name="text">
            <string>150 DPI</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>300 DPI</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>600 DPI</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1200 DPI</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Chart</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="4">
         <widget class="QLineEdit" name="chartTitle"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Font:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="chartFont">
          <item>
           <property name="text">
            <string>Arial</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Courrier</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Times</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="chartFontSize">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>6</number>
          </property>
          <property name="maximum">
           <number>64</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QToolButton" name="chartFontItalic">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>I</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QToolButton" name="chartFontBold">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>B</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Axes</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Axis X title:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="axisXTitle"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Axis Y title:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="axisYTitle"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Legend</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Location:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QComboBox" name="legendLocationComboBox">
          <item>
           <property name="text">
            <string>Right top</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right bottom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Left top</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Left bottom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Center top</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Center bottom</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="legendVisibleCheckBox">
          <property name="text">
           <string>Visible</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="unitDisplayedCheckBox">
          <property name="text">
           <string>Unit displayed</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="ChartImagePreview" name="imagePreview" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChartImagePreview</class>
   <extends>QWidget</extends>
   <header>ChartImagePreview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>imageWidth</tabstop>
  <tabstop>imageWidthUnit</tabstop>
  <tabstop>imageHeight</tabstop>
  <tabstop>imageHeightUnit</tabstop>
  <tabstop>imageResolution</tabstop>
  <tabstop>chartTitle</tabstop>
  <tabstop>chartFont</tabstop>
  <tabstop>chartFontSize</tabstop>
  <tabstop>chartFontBold</tabstop>
  <tabstop>chartFontItalic</tabstop>
  <tabstop>axisXTitle</tabstop>
  <tabstop>axisYTitle</tabstop>
  <tabstop>legendVisibleCheckBox</tabstop>
  <tabstop>unitDisplayedCheckBox</tabstop>
  <tabstop>legendLocationComboBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChartExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChartExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
